chpl/chpl-service/src/main/java/gov/healthit/chpl/util/ValidationUtils.java

```diff
@@ -197,15 +213,21 @@ public static CertificationCriterion getCert(String certNumber, List<Certificati
     }
 
     /**
-     * Look for required complimentary criteria; if any one of the
-     * criterionToCheck is present in allCriteriaMet then all of the
-     * complimentaryCertNumbers must be present in allCriteriaMet.
+     * DEPLRECATED - Look for required complimentary criteria; if any one of the criterionToCheck is present in
```

Comment:
The only actual change in this commit is adding the `@Deprecated` tag to this method, right? Can we just delete all of the javadoc stuff? Do they actually provide value? I'm seeing so many changes that are literally just whitespace changes I'd rather just have these things removed